nomad/skaffold.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: skaffold/v4beta11
+kind: Config
+metadata:
+  name: csi-driver
+build:
+  artifacts:
+    - image: localhost:30666/hetznercloud/hcloud-csi-driver
+      docker:
+        dockerfile: dev/Dockerfile
+      hooks:
+        after:
+          - command: ["sh", "-c", "./nomad/start_nomad.sh"]
```

Comment:
The `-c` flag stands for:

```
           -c               Read commands from the command_string operand instead of from the standard input.  Special parameter 0 will be set from the command_name operand and the positional parameters ($1, $2, etc.)  set from the remaining argu‐
                            ment operands.
```

So if you are launching a script, you only need: `["sh", "./nomad/start_nomad.sh"]`

Reply:
Had some trouble with the combination of `#!/usr/bin/env bash`, `sh` and `set -euo pipefail`. Switched now to `bash ./nomad/start_nomad.sh` and everything works now.

Maybe you wanna try again? :D

Did you run it from the root dir of the repo (`skaffold -f nomad/skaffold.yaml build`)?

Reply:
All works now for me, awesome!